src/commands/MigrationRunCommand.ts

```diff
@@ -46,6 +50,7 @@ export class MigrationRunCommand implements yargs.CommandModule {
                 subscribers: [],
                 synchronize: false,
                 migrationsRun: false,
+                migrationsForce: args["force"] ? true : connectionOptions.migrationsForce,
```

Comment:
probably its better to check if its `args["force"]  === "1" || args["force"] === "true"`

Reply:
Would setting the type to `boolean` suffice instead?

```
> require('yargs').option('force', { default: false, type: 'boolean' }).parse(['--force']).force
true
> require('yargs').option('force', { default: false, type: 'boolean' }).parse(['--force', 'false']).force
false
```

The only difference is that any other argument other than `"false"` would be recognized as `true`, so `--force 0` would evaluate to true. Are there any other options where `--option 0` is already being treated as false?

Reply:
thats okay

Reply:
Looks like an hour later you wrote the opposite in your review for `MigrationRevertCommand`:

> probably its better to check if its args["force"] === "1" || args["force"] === "true"

But in this case simply specifying `--force` in the CLI would not set the force arg to true. One would have to write either `--force true` or `--force 1`. I can make it that way, just think it might be confusing, since usually specifying a boolean `--flag` enables the flag in most CLI utilities I've used so far.

This is the way parsing currently works:

```
> var yargs = require('yargs').option('force', { default: false, type: 'boolean' })
undefined
> yargs.parse(['--force', 'true']).force
true
> yargs.parse(['--force', 'false']).force
false
> yargs.parse(['--force', '1']).force
true
> yargs.parse(['--force', '0']).force
true
```